cypress/plugins/index.js

```diff
@@ -0,0 +1,24 @@
+const wp = require('@cypress/webpack-preprocessor');
+module.exports = on => {
+  const options = {
+    webpackOptions: {
+      resolve: {
+        extensions: ['.ts', '.tsx', '.js'],
+      },
+      module: {
+        rules: [
+          {
+            test: /\.tsx?$/,
+            use: [
+              {
+                loader: 'ts-loader',
+                options: {transpileOnly: true},
```

Comment:
Cypress supports TS files by default, but doesn't support `imports` in spec files by default. This plugin adds support for imports by wrapping them with Webpack. Essentially every `spec` file is an entry point and a `script` tag that gets inserted into the test page. Only helper files should be imported so that helper code gets added as well. Another option could be that the helper folder gets added for every test automatically by adding it to `cypress/support/index.ts`, but that feels a bit too magical and may break "Jump to definition". We could try it...